ts/parts/Utilities.ts

```diff
@@ -3380,6 +3385,16 @@ H.isFunction = function (obj: unknown): obj is Function {
     return typeof obj === 'function';
 };
 
+H.isInsidePane = function (
+    x: number,
+    y: number,
+    center: Array<number>
+): boolean {
+    return Math.sqrt(
+        Math.pow(x - center[0], 2) + Math.pow(y - center[1], 2)
+    ) < center[2] / 2;
+};
+
```

Comment:
Can we move this function to `Pane.ts` file?

Reply:
Yes. Should I leave this function hanging in the pane module's scope? Also, I would have to move 2 events from `Polar.ts` to `Pane.ts` that use this function. What do you think about moving all the new code (all 3 events) from `Polar.ts` to `Pane.ts`?

Reply:
Do you mean `beforeGetHoverData` and `afterGetHoverData`? I'm fine with that.